Makefile

```diff
@@ -97,7 +103,7 @@ test:
 .PHONY: docker
 docker:
 	docker build -f docker/Dockerfile -t nknorg/nkn:latest-amd64 .
-	docker build -f docker/Dockerfile --build-arg build_args="build GOOS=linux GOARCH=arm BUILD_DIR=. BIN_DIR=." --build-arg base="arm32v6/" -t nknorg/nkn:latest-arm32v6 .
+	docker build -f docker/Dockerfile --build-arg build_args="build GOOS=linux GOARCH=arm GOARM=6 BUILD_DIR=. BIN_DIR=." --build-arg base="arm32v6/" -t nknorg/nkn:latest-arm32v6 .
 	docker build -f docker/Dockerfile --build-arg build_args="build GOOS=linux GOARCH=arm64 BUILD_DIR=. BIN_DIR=." --build-arg base="arm64v8/" -t nknorg/nkn:latest-arm64v8 .
```

Comment:
Should we add `GOARM=8` here also?

Reply:
No need, see: https://github.com/golang/go/wiki/GoArm